src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -40,25 +43,32 @@ public void init() {
         registerEndgameItems();
         registerCarbonStuff();
         registerSolarPanels();
+        registerRubber();
     }
 
     private void registerTools() {
         new CargoConfigurator().register(LiteXpansion.getInstance());
+        new Wrench().register(LiteXpansion.getInstance());
     }
 
     private void registerMachines() {
         new FoodSynthesizer().register(LiteXpansion.getInstance());
 
         new ScrapMachine().register(LiteXpansion.getInstance());
         new MassFabricator().register(LiteXpansion.getInstance());
-        new RubberSynthesizer().register(LiteXpansion.getInstance());
         new RefinedSmeltery().register(LiteXpansion.getInstance());
     }
 
-    private void registerMiscItems() {
-        //Rubber
-        registerNonPlaceableItem(Items.RUBBER, RubberSynthesizer.RECIPE_TYPE, SlimefunItems.OIL_BUCKET);
+    //Disable when SlimyTreeTaps exists
+    private void registerRubber() {
+        if (getServer().getPluginManager().getPlugin("SlimyTreeTaps") == null) {
```

Comment:
You will also need to add this as a `softdepend` to the plugin.yml, otherwise LX might load before Tree Taps.